apps/apps-backend/src/features/features.controller.ts

```diff
@@ -80,6 +80,9 @@ export class FeaturesController {
                         [Network.Custom]: false,
                     },
                 },
+                [Feature.FixedGasPrice]: {
```

Comment:
we have to make this feature just like `FiatConversion` to be able to enable it on certain networks only, devnet only will come first probably. Also, the feature must be disabled for all networks for now until devnet is upgraded to protocol v5